ipatests/test_integration/test_smb.py

```diff
@@ -194,6 +195,31 @@ def smb_sanity_check(self, user, client_mountpoint, share):
         finally:
             run_smb_server(['rm', '-rf', test_dir], raiseonerr=False)
 
+    def smb_installation_check(self, result):
+        domain_regexp_tpl = r'''
+            Domain\ name:\s*{domain}\n
+            \s*NetBIOS\ name:\s*{netbios}\n
+            \s*SID:\s+S-1-5-\d+-\d+-\d+-\d+\n
```

Comment:
Please check for `S-1-5-21-<domain>` here. (21 is important).

Reply:
fixed